compare-coverage.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+import subprocess
+
+def create_parser():
+    """Command line parser."""
+
+    parser = argparse.ArgumentParser(
+        description='Compare CBMC coverage checking results.'
+    )
+
+    parser.add_argument(
+        'coverage1',
+        help="""
+        JSON coverage output of make-coverage or cbmc-viewer.
+        """
+    )
+    parser.add_argument(
+        'coverage2',
+        help="""
+        JSON coverage output of make-coverage or cbmc-viewer.
+        """
+    )
+    return parser
+
+def dict_diff(dict1, dict2):
```

Comment:
It was a little unclear what this function does - took me several reads to understand. A comment would be helpful.